firebase-vscode/src/data-connect/language-client.ts

```diff
@@ -95,14 +95,30 @@ export function setupLanguageClient(
     // TODO: Expand to multiple services
     const config = configs.values[0];
     const generatedPath = ".dataconnect";
-    const schemaPaths = [
+
+    let schemaPaths = [
```

Comment:
Ahhh, I should have caught this in a previous review, but we should be using [path.join](https://nodejs.org/api/path.html#pathjoinpaths) whenever we create paths to avoid issues like this where we have to manually replace the delimiter.